Index: .dart_tool/chrome-device/Default/Extension Scripts/LOG.old
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>2022/02/01-00:30:01.734 10803 Reusing MANIFEST /private/var/folders/zk/z30tk_l54t92lz4g0fx0kcfc0000gn/T/flutter_tools.U5T4A1/flutter_tools_chrome_device.FwFEgY/Default/Extension Scripts/MANIFEST-000001\n2022/02/01-00:30:01.738 10803 Recovering log #3\n2022/02/01-00:30:01.741 10803 Reusing old log /private/var/folders/zk/z30tk_l54t92lz4g0fx0kcfc0000gn/T/flutter_tools.U5T4A1/flutter_tools_chrome_device.FwFEgY/Default/Extension Scripts/000003.log \n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .dart_tool/chrome-device/Default/Extension Scripts/LOG.old	(revision 7d920fc75ddb1b7264c487b99e221ab03803931d)
+++ .dart_tool/chrome-device/Default/Extension Scripts/LOG.old	(date 1643695095959)
@@ -1,3 +1,3 @@
-2022/02/01-00:30:01.734 10803 Reusing MANIFEST /private/var/folders/zk/z30tk_l54t92lz4g0fx0kcfc0000gn/T/flutter_tools.U5T4A1/flutter_tools_chrome_device.FwFEgY/Default/Extension Scripts/MANIFEST-000001
-2022/02/01-00:30:01.738 10803 Recovering log #3
-2022/02/01-00:30:01.741 10803 Reusing old log /private/var/folders/zk/z30tk_l54t92lz4g0fx0kcfc0000gn/T/flutter_tools.U5T4A1/flutter_tools_chrome_device.FwFEgY/Default/Extension Scripts/000003.log 
+2022/01/29-12:38:10.453 14103 Reusing MANIFEST /private/var/folders/6q/js4lgv250jxbfqc7pbq_ww840000gn/T/flutter_tools.CC8wHD/flutter_tools_chrome_device.BdICZn/Default/Extension Scripts/MANIFEST-000001
+2022/01/29-12:38:10.454 14103 Recovering log #3
+2022/01/29-12:38:10.454 14103 Reusing old log /private/var/folders/6q/js4lgv250jxbfqc7pbq_ww840000gn/T/flutter_tools.CC8wHD/flutter_tools_chrome_device.BdICZn/Default/Extension Scripts/000003.log 
